includes/formFactory/ManageWikiFormFactoryBuilder.php

```diff
@@ -197,13 +198,15 @@ private static function buildDescriptorExtensions(
 			if ( $ext['requires'] ) {
 				$requires = [];
 				foreach ( $ext['requires'] as $require => $data ) {
-					foreach ( $data as $index => $element ) {
-						if ( is_array( $element ) ) {
-							$data[$index] = '( ' . implode( ' OR ', $element ) . ' )';
+					if ( is_array( $data ) ) {
```

Comment:
When is $data not an array?

Reply:
$data is not an array when using 'articles' => $num, 'pages' => $num, and now 'visibility' => 'public/private'